portal-frontend/src/styles/functions.scss

```diff
@@ -0,0 +1,5 @@
+@use 'sass:math';
+
+@function rem($pixels, $context: 16) {
```

Comment:
Do I want to ask why? lol

Reply:
All the sizing in designs is in pixels. Also, the base size was specified as 18px (which does not make sense since the browser default is 16px). Anyway, I thought this way it should be easier for us to adjust sizes in the long run and at the same time the values will match pixels.